tests/testthat/test-normalize_exclusions.R

```diff
@@ -18,61 +18,88 @@ test_that("it merges two NULL or empty objects as an empty list", {
 })
 
 test_that("it returns the object if the other is NULL", {
-  t1 <- list(); t1[[a]] <- list(1:10)
+  t1 <- list()
```

Comment:
i guess `t1 = list(a = list(1:10))` would be better?

Reply:
NB `a <- tempfile()` so the list is named by a filename, not by literal `"a"`

Reply:
oh sorry, silly mistake. yea setNames is not much different in that case.